rollup.config.mjs

```diff
@@ -0,0 +1,94 @@
+import commonjs from '@rollup/plugin-commonjs';
+import resolve from '@rollup/plugin-node-resolve';
+import terser from '@rollup/plugin-terser';
+import typescript from '@rollup/plugin-typescript';
+import url from '@rollup/plugin-url';
+import svgr from '@svgr/rollup';
+import { defineConfig } from 'rollup';
+import dts from 'rollup-plugin-dts';
+import peerDepsExternal from 'rollup-plugin-peer-deps-external';
+import postcss from 'rollup-plugin-postcss';
+import preserveDirectives from 'rollup-plugin-preserve-directives';
+import progress from 'rollup-plugin-progress';
+import { typescriptPaths } from 'rollup-plugin-typescript-paths';
+import { visualizer } from 'rollup-plugin-visualizer';
+
+export default defineConfig(
+  {
+    input: ['src/octuple.ts', 'src/locale.ts'],
+    output: [
+      {
+        chunkFileNames: '[name]-[hash].cjs',
+        dir: 'lib',
+        entryFileNames: '[name].cjs',
+        format: 'cjs',
+        sourcemap: false,
+      },
+      {
+        chunkFileNames: '[name]-[hash].mjs',
+        dir: 'lib',
+        entryFileNames: '[name].mjs',
+        format: 'es',
+        sourcemap: false,
+      },
+    ],
+    plugins: [
+      progress(),
+      postcss({
+        modules: { localsConvention: 'camelCase' },
```

Comment:
Did you run the UTs locally, they may use webpack still, but should be verified.

Reply:
Tests seemed to all pass when running `yarn test`
![Screenshot 2024-05-09 at 5 55 55 PM](https://github.com/EightfoldAI/octuple/assets/87032295/4fc00be5-4128-4f80-9830-0b8146f689ff)